src/hotspot/share/ci/ciInstanceKlass.cpp

```diff
@@ -471,11 +471,6 @@ ciField* ciInstanceKlass::get_field_by_name(ciSymbol* name, ciSymbol* signature,
   return field;
 }
 
-static int sort_field_by_offset(ciField** a, ciField** b) {
```

Comment:
Could you add a comment at `ciInstanceKlass._declared_nonstatic_fields` and `_nonstatic_fields` to specify exactly the order of the fields there? It is a little bit hard to grasp at the first sight, especially with flattened fields.